src/xcode/ENA/ENA/Source/Scenes/DataDonation/Model/__tests__/DataDonationModelTests.swift

```diff
@@ -10,10 +10,11 @@ class DataDonationModelTests: XCTestCase {
 	func testGIVEN_ModelWithJson_WHEN_GetValues_THEN_ValuesAreEqualToDefault() throws {
 		// GIVEN
 		let mockStore = MockTestStore()
-
+		mockStore.isPrivacyPreservingAnalyticsConsentGiven = false
```

Comment:
It would be eve nicer if this is the default setting. We could then check on new (Mock-)Stores if this is is the case:
```
let mockStore = MockTestStore()
XCTAssertFalse(mockStore. isPrivacyPreservingAnalyticsConsentGiven)
```

Reply:
Update: I've seen the default value for the real store. Having two different default values might lead to confusion and invalid tests due to the different behaviour of the stores.

Reply:
Yeah, you are right. I changed the default value to false. Unfortunately, many tests must declare it now explicitly.